### PR TITLE
show only Google group is google_maps feature flag is enabled

### PR DESCRIPTION
**Pull request by javisantana**
cc @xavijam 

this makes that only google basemaps are shown when google_maps feature is enabled

### CODE REVIEW

**Comment by xavijam** [comment]
:+1:

File: lib/assets/javascripts/cartodb/table/basemap/basemap_dropdown/basemap_dropdown.js [hunk 1]
```diff
@@ -139,7 +139,13 @@ cdb.admin.DropdownBasemap = cdb.ui.common.Dropdown.extend({
         return;
       }
     } else {
-       v = new cdb.admin.BaseMapView({ model: lyr, map: this.model });
+       // this is a temporal fix, if google maps feature flag is enabled show only maps in group google maps
+      if (this.user.featureEnabled('google_maps')) {
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/table/basemap/basemap_dropdown/basemap_dropdown.js]: Why not both conditions in the same line?

Comment (javisantana) [inline lib/assets/javascripts/cartodb/table/basemap/basemap_dropdown/basemap_dropdown.js]: yep

Comment (javisantana) [inline lib/assets/javascripts/cartodb/table/basemap/basemap_dropdown/basemap_dropdown.js]: this is temporal code so is not that important

Comment (xavijam) [inline lib/assets/javascripts/cartodb/table/basemap/basemap_dropdown/basemap_dropdown.js]: agree

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3507/))

**Pull request merged**